packages/opentelemetry-api/src/trace/link.ts

```diff
@@ -24,7 +24,7 @@ import { SpanContext } from './span_context';
  */
 export interface Link {
   /** The {@link SpanContext} of a linked span. */
-  spanContext: SpanContext;
+  linkContext: LinkContext;
```

Comment:
should this be named just a `context` to avoid `link.linkContext` and have simply `link.context` ?

Reply:
Sounds good.